spot_wrapper/spot_arm.py

```diff
@@ -612,3 +613,49 @@ def grasp_3d(self, frame: str, object_rt_frame: typing.List[float]) -> typing.Tu
                 return False, msg
         except Exception as e:
             return False, f"An error occured while trying to grasp from pose {e}"
+
+    def override_grasp_or_carry(self, grasp_override: manipulation_api_pb2.ApiGraspOverride.Override,
```

Comment:
nit: maybe have these as `Optional`s so that you can pass in `None` if you don't care about overriding one of the states?

Reply:
You use the UNKNOWN (0) choice for that for both of them.

Reply:
I started with them as optional and then realized it was pretty annoying because I was converting from the protobuf types anyway...